bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/ui/wizard/CreateContractInputFromBusinessObjectWizardPage.java

```diff
@@ -308,25 +320,85 @@ protected MultiValidator createEmptySelectionMultivalidator(final IViewerObserva
             protected IStatus validate() {
                 if (checkedElements.isEmpty()) {
                     return ValidationStatus.error(Messages.atLeastOneAttributeShouldBeSelectedError);
+                } else {
```

Comment:
The anonymous class starts to be complicated, aybe extracted it to a specific class will simplify code and help to test easily